docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+        <div class="d-flex mb-4">
+          {% if data.currentPage > 1 %}
+          <a href="{{ data.slug }}?page={{data.currentPage - 1}}" class="btn btn-primary text-uppercase me-auto">newer posts</a>
+          {% endif %}
+          {% if data.totalPages > data.currentPage %}
+          <a href="{{ data.slug }}?page={{data.currentPage + 1}}" class="btn btn-primary text-uppercase ms-auto">older posts</a>
+          {% endif %}
+        </div>
+      </div>
+    </div>
+  </div>
+
+{% endblock %}
+```
+:::
+
+<template v-slot:caption>
+modules/blog-page/index.html
+</template>
+</AposCodeBlock>
+
+Focusing on the `for` loop in the code. We are stepping through all of the articles returned in `data.pieces` and outputing the relevant data. Again, since we specified a `perPage` value of `5` in the options, this will return the five newest blog articles. This can be further configured within the `blog-page` module options, for example with the [`sort` option](https://v3.docs.apostrophecms.org/reference/modules/piece-type.html#sort).
+
+The `Pager` section is expanded to conditionally show newer and older blog articles, unlike the original template, which only shows older articles. Within the section, we are taking advantage of some additional data that is being delivered to the `index.html` page. Within the `data` payload are `data.currentPage` and `data.totalPages`. The `data.totalPages` is how many individual data sets are present for the particular piece type if divided into groups based on the `perPage` option (the default is 10).
+
+By default, we are showing the newest blog articles first. Therefore, if the `data.currentPage` is equal to `1` then we shouldn't display the button to load newer articles. If we are on any other page we want the button displayed, with a URL that adds a query to go to the previous page -
+::: v-pre
+(`{{data.currentPage - 1}}`).
+:::
+
+We are only displaying the button to go to older posts if we aren't at the last set of pieces - (`data.totalPages > data.currentPage`). If this is true we display a button that points to the URL with a query that goes to the next set of pieces - 
+::: v-pre
+(`{{data.currentPage + 1}}`).
+:::
+
+### The blog show.html page
+
+The `show.html` page will display each of the individual blog articles and will be based on the original HTML template `post.html` page. Like the other pages, we start by rendering the header fragment in the `beforeMain` block. If we look at the `post.html` page we can see that the header looks slightly different from the other pages. It contains metadata not found on the other pages. After setting up the main part of the page we will alter the `header.html` fragment to address this.
+
+Open the `post.html` template file and copy the `Post Content` section into the `main` block of the `show.html` page. All of the content in `<p>` tags can be deleted because we will replace it with the content added to the `main` area of our blog pieces.
```

Comment:
```suggestion
Open the `post.html` template file and copy the "Post Content" section into the `main` block of the `show.html` page. All of the content in `p` tags can be deleted because we will replace it with the content added to the `main` area of our blog pieces.
```

Same formatting suggestion as above. Not sure you need the `< >` if you also say "tags".

Reply:
Changed.